src/transform.ts

```diff
@@ -111,6 +111,11 @@ export interface AggregatedFieldDef {
    */
   field?: FieldName;
 
+  /**
+   * A parameter that can be passed to aggregation functions. The aggregation operation `"exponential"` requires it.
+   */
+  aggregate_param?: number;
```

Comment:
`aggregate_params` can also be used in [window transforms](https://github.com/vega/vega/blob/main/packages/vega-transforms/src/Window.js#L41), but the type [`WindowTransform` is not updated](https://github.com/vega/vega/blob/main/packages/vega-typings/types/spec/transform.d.ts#L715). 

If I'm not misunderstanding something, I can open a pr to update `WindowTransform` so we can add exponential moving average to window transforms.

Reply:
Ah, good catch. Yes, please send a pull request and I can merge it. We can add support for window aggregates in a follow up pull request.

Reply:
Done! Opened https://github.com/vega/vega/pull/3874